designs/2022-docs-information-architecture-update/README.md

```diff
@@ -0,0 +1,352 @@
+  - Current page: [Contributing](https://eslint.org/docs/latest/developer-guide/contributing/)
+  - Hidden in sidebar
+- [MOVE & RENAME PAGE] Report Bugs
+  - Current page: [Bug Reporting](https://eslint.org/docs/latest/developer-guide/contributing/reporting-bugs)
+- [MOVE & RENAME PAGE] Propose a New Rule
+  - Current page: [Proposing a New Rule](https://eslint.org/docs/latest/developer-guide/contributing/new-rules)
+- [MOVE & RENAME PAGE] Propose a Rule Change
+  - Current page: [Proposing a Rule Change](https://eslint.org/docs/latest/developer-guide/contributing/#:~:text=Proposing%20a-,Rule%20Change,-Want%20to%20make)
+- [MOVE & RENAME PAGE] Request a Change
+  - Current page: [Change Requests](https://eslint.org/docs/latest/developer-guide/contributing/changes)
+- [MOVE & RENAME PAGE] Manage Issues
+  - Current page: [Managing Issues](https://eslint.org/docs/latest/maintainer-guide/issues)
+- [MOVE & RENAME PAGE] Review Pull Requests
+  - Current page: [Reviewing Pull Requests](https://eslint.org/docs/latest/maintainer-guide/pullrequests)
+- [MOVE & RENAME PAGE] Manage Releases
+  - Current page: [Managing Releases](https://eslint.org/docs/latest/maintainer-guide/releases)
+- [MOVE & RENAME PAGE] Work on Issues
+  - Current page: [Working on Issues](https://eslint.org/docs/latest/developer-guide/contributing/working-on-issues)
+- [MOVE & RENAME PAGE] Submit a Pull Request
+  - Current page: [Pull Requests](https://eslint.org/docs/latest/developer-guide/contributing/pull-requests)
+- [NEW PAGE] Report Security Vulnerabilities
+  - Tales current content from the [Reporting a Security Vulnerability section on the Contributing page](https://eslint.org/docs/latest/developer-guide/contributing/#reporting-a-security-vulnerability). Worth breaking out into its own page for discoverability and SEO for an important subject.
+  - NOTE: Should be added, but not part of the initial IA project. Probably can be done as a one-off task or group in with one of the project phases.
+- [MOVE PAGE] Governance
+  - Current page: [Governance](https://eslint.org/docs/latest/maintainer-guide/governance)
+
+### [REFACTOR SECTION] Maintain ESLint
+
+- [NO CHANGE] Maintain ESLint section landing page
+  - Current page: [Maintainer Guide](https://eslint.org/docs/latest/maintainer-guide)
+  - Hidden in sidebar
+- [MOVE PAGE] Architecture
+  - Current page: [Architecture](https://eslint.org/docs/latest/developer-guide/architecture/)
+  - Note: in a previous conversation with Nicholas, he mentioned to me that this page is pretty out-of date.
+  - There should probably be a future unit of work to refactor this page.
+  - However, it’s not included in the information architecture updates.
+- [MOVE & REFACTOR PAGE] Set up Development Environment
+  - Consolidate the pages [Getting the Source Code](https://eslint.org/docs/latest/developer-guide/source-code) and [Set up a Development Environment](https://eslint.org/docs/latest/developer-guide/development-environment)
```

Comment:
Good idea to combine these.

https://eslint.org/docs/latest/developer-guide/contributing/pull-requests links to dev environment setup, so should this combined page be part of the Contribute to ESLint section instead, either within or immediately after Submit a Pull Request?

*Edit: The sidebar summary below shows this page in the contributor guide. If that's been updated more recently, I agree :+1: I also noticed the location of the release guide is inconsistent.*

Reply:
looks like the sidebar summary was correct and things were added incorrectly over here. sorry for confusion. fixing now.